sdks/python/setup.py

```diff
@@ -182,7 +182,7 @@ def get_version():
     'freezegun>=0.3.12',
     'nose>=1.3.7',
     'nose_xunitmp>=0.4.1',
-    'pandas>=0.23.4,<0.25',
+    'pandas>=0.25.2,<1',
```

Comment:
Do you need py2/py3 separation similar to https://github.com/apache/beam/blob/9838377bf73c51bd43ca89a83c483ea91e226c29/sdks/python/container/base_image_requirements.txt#L66 ?

Reply:
Yes, in addition to that  pandas are also fussy about  py 3.5.2 which we still have on Jenkins.